doc/i2p.md

```diff
@@ -10,11 +10,22 @@ started with I2P terminology.
 ## Run Bitcoin Core with an I2P router (proxy)
 
 A running I2P router (proxy) with [SAM](https://geti2p.net/en/docs/api/samv3)
-enabled is required (there is an [official one](https://geti2p.net) and
-[a few alternatives](https://en.wikipedia.org/wiki/I2P#Routers)). Notice the IP
-address and port the SAM proxy is listening to; usually, it is
-`127.0.0.1:7656`. Once it is up and running with SAM enabled, use the following
-Bitcoin Core options:
+
+enabled is required. Options include:
+
+- [i2p-router](https://geti2p.net), the official implementation in Java
```

Comment:
Not sure if this implementation in Java should be mentioned because it doesn't help with Bitcoin Core setup

Reply:
I'm not sure what you mean. This was already in the doc, and I've heard from people who are using it with Bitcoin Core.

Reply:
https://github.com/bitcoin/bitcoin/pull/22250#discussion_r654495784

Reply:
Thanks for the info! ISTM the link should still be provided.

Reply:
I tested v22.0rc2 with the official Java i2prouter with no issues on Debian.  You have to follow the instructions at their download link, and then visit 127.0.0.1:7657 to start SAM.  You also need to type `i2prouter start` in terminal.

Reply:
127.0.0.1:7657 -> ConfigureHomepage -> Clients  then hit the Play Button next to "SAM application Bridge".  On the left side of the page, there should be a green light next to "Shared Clients".

Reply:
Thanks for the info, @Rspigler. Resolving here for now.